### ISSUE TITLE
Installation for gdm

### ISSUE THREAD

**Issue by AToMiXhawK** [xulongwu4/solus-nvidia-optimus-manager#2]
99-nvidia.conf: copy to /etc/lightdm/lightdm.conf.d/99-nvidia.conf

to where 99-nvidia.conf should be copied if using Solus Gnome with gdm

**Comment by kz6fittycent** [comment]
Yeah Budgie, MATE, Gnome all use `/etc/gdm` now. Not a big deal as it is just a simple directory change. I've submitted a PR with an automated installation script ;)
